docs/_source/conf.py

```diff
@@ -20,18 +20,20 @@
 import sys
 
 # Get the project root dir, which is the parent dir of this
-project_path = os.path.join(os.path.dirname(__file__), '..' + os.path.sep + '..')
+project_path = os.path.join(
+    os.path.dirname(__file__), ".." + os.path.sep + ".."
+)
 sys.path.insert(0, os.path.abspath(project_path))
 # -- Project information -----------------------------------------------------
 
-project = 'Sceptre'
-copyright = '2018, Cloudreach'
-author = 'Cloudreach'
+project = "Sceptre"
```

Comment:
The default and convention in the conf.py file is to use single quotes, can you please revert the changes that introduced the double quotes. Looks like a editor probably changed these automatically.